cf_xarray/tests/test_accessor.py

```diff
@@ -212,6 +177,7 @@ def test_preserve_unused_keys():
     assert_identical(actual, expected)
 
 
+@pytest.mark.skip(reason="xarray GH4120. any test after this will fail since attrs are lost")
```

Comment:
Sorry @jukent that was bad advice on my part. Let's just comment out the coarsen lines below and not skip this. We definitely want to check `isel` and `mean` for regressions

Reply:
You could do some advanced skipping using the xarray version but I don't think it's worth it...